app/ui/src/app/customizations/api-connector/api-connector-detail.component.ts

```diff
@@ -12,7 +12,7 @@ import { ApiConnectorStore } from './api-connector.store';
 })
 export class ApiConnectorDetailComponent implements OnInit {
   // TODO: Update Restangular/Store setup config to return a properly typed ApiConnectorData instance object
```

Comment:
Perhaps you'll want to remove this comment line, since it is no longer required.